autoload/neomake.vim

```diff
@@ -658,10 +665,12 @@ function! s:AddExprCallback(jobinfo, prev_index) abort
     let ignored_signs = 0
     let s:postprocess = get(maker, 'postprocess', function('neomake#utils#CompressWhitespace'))
     let debug = get(g:, 'neomake_verbose', 0) >= 3
+    let custom_qf = get(g:, '_neomake_qf_enabled', 0)
```

Comment:
In some sense, yes.  Although I prefer the former style, and we could move slowly towards it instead.